products/sle/main.pm

```diff
@@ -1118,38 +1118,31 @@ sub load_online_migration_tests {
 }
 
 sub load_patching_tests {
-    if (check_var("ARCH", "s390x")) {
-        if (check_var('BACKEND', 's390x')) {
-            loadtest "installation/bootloader_s390";
-        }
-        else {
-            loadtest "installation/bootloader_zkvm";
-        }
-    }
     # Switch to orginal system version for upgrade tests
     if (is_upgrade) {
         # Save HDDVERSION to ORIGIN_SYSTEM_VERSION
         set_var('ORIGIN_SYSTEM_VERSION', get_var('HDDVERSION'));
         # Save VERSION to UPGRADE_TARGET_VERSION
         set_var('UPGRADE_TARGET_VERSION', get_var('VERSION'));
+        # Always boot from installer DVD in upgrade test
+        set_var('BOOTFROM', 'd');
```

Comment:
Don't we want to set this variable in the test suite?

Reply:
Interesting question. We have some variables set inside tests and some in test suite. Not saying that you are wrong but how you would define a rule which variables should be set where ?

Reply:
Yes, we can also set the variables in test suite, and it actually is what we are doing currently: for every upgrade test suite, we set both BOOTFROM=d and BOOT_HDD_IMAGE=1, which means boot from DVD but boot into harddisk for patching original system.
I try to set these two variables here to avoid adding them in tens of upgrade test suite.
Another reason is that it is a little confusing at the first glance to see both BOOTFROM=d and BOOT_HDD_IMAGE=1 variables in a single test suite, I hope setting them in code with a comment is more comprehensible.